atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -501,8 +501,9 @@ def add_log_type(output):
             output_files.append(output_file)
 
         # add remote source tarball and remote-source.json files to output
+        nvr = '{}-{}-{}'.format(build['name'], build['version'], build['release'])
```

Comment:
Am I missing any cases where any of the NVR components would not be available at this point?

Reply:
get_build will always return nvr so should be fine